pages/docs/linkfree-profile-tips.mdx

```diff
@@ -0,0 +1,21 @@
+import DocsLayout from "@components/layouts/DocsLayout.js"; import Link from "@components/Link";
```

Comment:
I think these should be on separate lines, if you us the Prettier VScode plugin all the formatting will be done for you

Reply:
Ok